### DISCUSSION TITLE
Account suspend

### DISCUSSION THREAD

**Discussion by exshapoval** [community/community#84127] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi

Account devsquadcore was suspended and we do not understand why.
We do not receive email with explain
We can not login to support.github.com
We send message via https://support.github.com/contact/cannot_sign_in and nothing

Please, help

**Comment by queenofcorgis** [comment]
Hi @exshapoval, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
